src/layouts/layout.jsx

```diff
@@ -120,26 +121,38 @@ function Layout({ children }) {
   return (
     <>
       <AppContext.Provider value={{ state: contextState, setState: setContextState }}>
+        {showTypeForm &&
```

Comment:
deja vu? It should be a component so you DRY

Reply:
I don't understand

Reply:
this code is repeated in `mobile-menu`, why don't you create a component for it so you don't repeat yourself (DRY)

Reply:
Done